projects/core/src/global-message/store/global-message-store.module.ts

```diff
@@ -2,8 +2,12 @@ import { NgModule } from '@angular/core';
 import { StoreModule } from '@ngrx/store';
 import { reducerToken, reducerProvider } from './reducers/index';
```

Comment:
import { reducerToken, reducerProvider, GLOBAL_MESSAGE_FEATURE } from './index';

Reply:
I prefer to be more specific regarding imports